### PR TITLE
Fix formatParameter() for boolean

### PR DESCRIPTION
**Pull request by jderusse**
|      Q       |   A
|------------- | -----------
| Type         | bug
| BC Break     | yes
| Fixed issues | n/a

#### Summary

This PR fixes BC break when formatting a boolean parameters (and type is not defined)

```
        $this->addSql('INSERT INTO XXX (..., is_bar) VALUES (..., :is_bar)', [
            'is_bar' => true,
        ]);
```

### CODE REVIEW

File: lib/Doctrine/Migrations/InlineParameterFormatter.php [hunk 1]
```diff
@@ -53,7 +53,7 @@ public function formatParameters(array $params, array $types): string
         return sprintf('with parameters (%s)', implode(', ', $formattedParameters));
     }
 
-    private function formatParameter(mixed $value, string|int $type): string|int|float|null
+    private function formatParameter(mixed $value, string|int $type): string|int|bool|float|null
```
Comment (stof) [inline lib/Doctrine/Migrations/InlineParameterFormatter.php:56]: Why this change ? `parameterToString` returns a string.

Comment (jderusse) [reply lib/Doctrine/Migrations/InlineParameterFormatter.php:56]: `parameterToString` does, but `Type::getType('string')->convertToDatabaseValue(...)` don't

**Comment by jderusse** [comment]
friendly reminder

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thank you!
